tests/integ/attr_test.py

```diff
@@ -249,9 +248,8 @@ def testObjAttr(self):
             # try adding the attribute as a different user
             user2_name = config.get("user2_name")
             if user2_name:
-                headers = helper.getRequestHeaders(
-                    domain=self.base_domain, username="test_user2"
-                )
+                headers = helper.getRequestHeaders(domain=self.base_domain, username="test_user2")
+                attr_payload = {"type": "H5T_STD_I32LE", "value": 42}
```

Comment:
This test fails due to `attr_payload` being undefined if `user2_name` isn't found in the config - this should either have a default second username, or the requests should only be made if a second username is found.

Reply:
Fixed